processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/load_use.asm (1 of 1)
--- memsim.hex	2021-10-05 23:57:58.653591765 -0400
+++ memcpu.hex	2021-10-05 23:57:58.578588925 -0400
@@ -7,5 +7,6 @@
 :04000600AC23000423
 :040007002063111150
 :04000800FFFFFFFFF8
-:04003C00000011119E
+:04003C00BAD1CBE288
+:04003D00BAD1BAD1A9
 :00000001FF
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-06 00:25:29.766105273 -0400
+++ memcpu.hex	2021-10-06 00:25:29.698102696 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:25:32.793219892 -0400
+++ memcpu.hex	2021-10-06 00:25:32.695216181 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:29:05.364269308 -0400
+++ memcpu.hex	2021-10-06 00:29:05.286266355 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-06 00:29:20.211831544 -0400
+++ memcpu.hex	2021-10-06 00:29:20.132828552 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:29:23.368951095 -0400
+++ memcpu.hex	2021-10-06 00:29:23.294948293 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:30:19.894091544 -0400
+++ memcpu.hex	2021-10-06 00:30:19.829089083 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:30:45.853074538 -0400
+++ memcpu.hex	2021-10-06 00:30:45.769071357 -0400
@@ -1,3 +1,4 @@
+:040000008FA30000CA
 :0400010027BD000413
 :0400020034040001C1
 :0400030034060000BF
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:31:30.886779838 -0400
+++ memcpu.hex	2021-10-06 00:31:30.773775559 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:32:30.110022457 -0400
+++ memcpu.hex	2021-10-06 00:32:30.035019617 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:34:55.675534615 -0400
+++ memcpu.hex	2021-10-06 00:34:55.603531888 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:35:23.554590317 -0400
+++ memcpu.hex	2021-10-06 00:35:23.477587401 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/a.asm (1 of 1)
--- memsim.hex	2021-10-06 00:43:36.439254445 -0400
+++ memcpu.hex	2021-10-06 00:43:36.371251870 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/a.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:44:18.638852360 -0400
+++ memcpu.hex	2021-10-06 00:44:18.583850278 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-06 00:44:27.089172337 -0400
+++ memcpu.hex	2021-10-06 00:44:27.031170141 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:44:29.151250419 -0400
+++ memcpu.hex	2021-10-06 00:44:29.098248412 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:44:51.099081488 -0400
+++ memcpu.hex	2021-10-06 00:44:51.019078458 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:45:21.311225490 -0400
+++ memcpu.hex	2021-10-06 00:45:21.239222764 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-06 00:45:41.450988095 -0400
+++ memcpu.hex	2021-10-06 00:45:41.379985407 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 00:45:43.940082346 -0400
+++ memcpu.hex	2021-10-06 00:45:43.869079658 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC0000001900A8
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/jump_hazard.asm (1 of 1)
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 01:04:10.500985601 -0400
+++ memcpu.hex	2021-10-06 01:04:10.429982912 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-06 01:04:58.284795508 -0400
+++ memcpu.hex	2021-10-06 01:04:58.229793424 -0400
@@ -1,4 +1,4 @@
-:040001008FA20000CA
+:04000000341DFFFCB0
 :0400020027BD000412
 :040003008FA30000C7
 :0400040027BD000410
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 01:05:01.184905355 -0400
+++ memcpu.hex	2021-10-06 01:05:01.126903158 -0400
@@ -1,4 +1,3 @@
-:04000000341DFFFCB0
 :040001003402001FA6
 :040002003403000AB9
 :0400030034040010B1
@@ -31,6 +30,5 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFD000000000AB6
 :00000001FF
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2021-10-06 01:06:23.740032294 -0400
+++ memcpu.hex	2021-10-06 01:06:23.650028885 -0400
@@ -81,133 +81,102 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:04014000000000308B
+:0401410000000008B2
+:040142000000004277
+:04015900000000643E
+:04015A000000000F92
+:04015B000000005C44
+:04015C00000000415E
+:04015D00000000207E
+:04015E000000000994
+:04015F000000002F6D
+:040160000000005942
+:040161000000001189
+:040162000000000792
+:040163000000002375
+:040164000000004453
+:040165000000002076
+:040166000000000A8B
+:04016700000000078D
+:04016800000000177C
+:040169000000005C36
+:04016A000000005B36
+:04016B000000002868
+:04016C000000001A75
+:04016D000000000886
+:04016E000000002469
+:04016F000000002666
+:040170000000000883
+:040171000000002664
+:040172000000001079
+:040173000000003256
+:040174000000000780
+:040175000000004343
+:040176000000006421
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2021-10-06 01:06:36.372510774 -0400
+++ memcpu.hex	2021-10-06 01:06:36.286507516 -0400
@@ -81,133 +81,102 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:04014000000000308B
+:0401410000000008B2
+:040142000000004277
+:04015900000000643E
+:04015A000000000F92
+:04015B000000005C44
+:04015C00000000415E
+:04015D00000000207E
+:04015E000000000994
+:04015F000000002F6D
+:040160000000005942
+:040161000000001189
+:040162000000000792
+:040163000000002375
+:040164000000004453
+:040165000000002076
+:040166000000000A8B
+:04016700000000078D
+:04016800000000177C
+:040169000000005C36
+:04016A000000005B36
+:04016B000000002868
+:04016C000000001A75
+:04016D000000000886
+:04016E000000002469
+:04016F000000002666
+:040170000000000883
+:040171000000002664
+:040172000000001079
+:040173000000003256
+:040174000000000780
+:040175000000004343
+:040176000000006421
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2021-10-06 01:07:26.902424694 -0400
+++ memcpu.hex	2021-10-06 01:07:26.836422195 -0400
@@ -81,133 +81,102 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:04014000000000308B
+:0401410000000008B2
+:040142000000004277
+:04015900000000643E
+:04015A000000000F92
+:04015B000000005C44
+:04015C00000000415E
+:04015D00000000207E
+:04015E000000000994
+:04015F000000002F6D
+:040160000000005942
+:040161000000001189
+:040162000000000792
+:040163000000002375
+:040164000000004453
+:040165000000002076
+:040166000000000A8B
+:04016700000000078D
+:04016800000000177C
+:040169000000005C36
+:04016A000000005B36
+:04016B000000002868
+:04016C000000001A75
+:04016D000000000886
+:04016E000000002469
+:04016F000000002666
+:040170000000000883
+:040171000000002664
+:040172000000001079
+:040173000000003256
+:040174000000000780
+:040175000000004343
+:040176000000006421
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2021-10-06 01:07:37.762836054 -0400
+++ memcpu.hex	2021-10-06 01:07:37.688833251 -0400
@@ -81,133 +81,102 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:04014000000000308B
+:0401410000000008B2
+:040142000000004277
+:04015900000000643E
+:04015A000000000F92
+:04015B000000005C44
+:04015C00000000415E
+:04015D00000000207E
+:04015E000000000994
+:04015F000000002F6D
+:040160000000005942
+:040161000000001189
+:040162000000000792
+:040163000000002375
+:040164000000004453
+:040165000000002076
+:040166000000000A8B
+:04016700000000078D
+:04016800000000177C
+:040169000000005C36
+:04016A000000005B36
+:04016B000000002868
+:04016C000000001A75
+:04016D000000000886
+:04016E000000002469
+:04016F000000002666
+:040170000000000883
+:040171000000002664
+:040172000000001079
+:040173000000003256
+:040174000000000780
+:040175000000004343
+:040176000000006421
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-06 01:07:47.130190861 -0400
+++ memcpu.hex	2021-10-06 01:07:47.057188096 -0400
@@ -1,4 +1,4 @@
-:040001008FA20000CA
+:04000000341DFFFCB0
 :0400020027BD000412
 :040003008FA30000C7
 :0400040027BD000410
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-06 01:07:50.236308512 -0400
+++ memcpu.hex	2021-10-06 01:07:50.159305595 -0400
@@ -1,4 +1,3 @@
-:04000000341DFFFCB0
 :040001003402001FA6
 :040002003403000AB9
 :0400030034040010B1
@@ -31,6 +30,5 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFD000000000AB6
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/beq_not_taken.asm (1 of 23)
processing file asmFiles/beq_taken.asm (2 of 23)
processing file asmFiles/bne_not_taken.asm (3 of 23)
processing file asmFiles/bne_taken.asm (4 of 23)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 1)
